src/current/v23.2/set-up-physical-cluster-replication.md

```diff
@@ -0,0 +1,388 @@
+
+### Connect to the primary virtual cluster (optional)
+
+1. If you would like to run a sample workload on the primary's application virtual cluster, open a new terminal window and use [`cockroach workload`]({% link {{ page.version.version }}/cockroach-workload.md %}) to run the workload.
+
+    For example, to initiate the [`movr`]({% link {{ page.version.version }}/movr.md %}) workload:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ shell
+    cockroach workload init movr "postgresql://root@{node IP or hostname}:{26257}/?options=-ccluster=application&sslmode=verify-full&sslrootcert=certs/ca.crt&sslcert=certs/client.root.crt&sslkey=certs/client.root.key"
+    ~~~
+
+    {% include {{ page.version.version }}/connect/cockroach-workload-parameters.md %} As a result, for the example in this tutorial, you will need:
+    - `options=-ccluster=application`
+    - `sslmode=verify-full`
+    - `sslrootcert={path}/certs/ca.crt`: the path to the CA certifcate.
+    - `sslcert={path}/certs/client.root.crt`: the path to the client certificate.
+    - `sslkey={path}/certs/client.root.key`: the path to the client private key.
+
+    For additional detail on the standard CockroachDB connection parameters, refer to [Client Connection Parameters]({% link {{ page.version.version }}/connection-parameters.md %}#connect-using-a-url).
+
+1. Run the `movr` workload for a set duration using the same connection string:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ shell
+    cockroach workload run movr --duration=5m "postgresql://root@{node IP or hostname}:{26257}/?options=-ccluster=application&sslmode=verify-full&sslrootcert=certs/ca.crt&sslcert=certs/client.root.crt&sslkey=certs/client.root.key"
+    ~~~
+
+1. To connect to the primary's application virtual cluster, use the `ccluster=application` parameter:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ shell
+    cockroach sql --url \
+    "postgresql://root@{node IP or hostname}:26257/?options=-ccluster=application&sslmode=verify-full" \
+    --certs-dir "certs"
+    ~~~
+
+    Your prompt will include `application` when you have successfully connected.
+
+1. Create a user for your primary's application cluster:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ sql
+    CREATE USER {your username} WITH PASSWORD '{your password}';
+    ~~~
+
+1. You can connect to the [DB Console]({% link {{ page.version.version }}/ui-overview.md %}) with this user to observe activity on the primary cluster. Open a web browser at `https://{node IP or hostname}:8080/` and enter your credentials.
+
+## Step 2. Create the standby cluster
+
+### Start the standby cluster
+
+Similarly to the primary cluster, it is necessary to start each node with the `--config-profile` flag and a `replication-target` for the standby cluster. This creates a _virtualized cluster_ with a system interface, and sets up all the required configuration for starting a replication stream.
```

Comment:
```suggestion
Similarly to the primary cluster,each node on the standby cluster must be started with the `--config-profile` flag set to `replication-target`. This creates a _virtualized cluster_ with a system interface and an application cluster, and sets up all the required configuration for starting a replication stream.
```

Reply:
Nice, I like this, changed!

Reply:
For followup, I don't think that the standby cluster actually gets an auto-created virtual cluster. You specifically walk them through creating it.